packages/api-integration/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../tsconfig.base",
+  "compilerOptions": {
+    "outDir": "./dist",
+    "rootDir": "./src",
+    "lib": ["es6", "dom"],
```

Comment:
Don't think we need this line? We need it in packages which need the web API types.